### ISSUE TITLE
Posh-SSH cannot connect to AudioCodes Mediant 2600

### ISSUE THREAD

**Issue by finnslossdc** [darkoperator/Posh-SSH#522]
I'm trying to connect to an AudioCodes Mediant 2600 device, but Posh-SSH just times out.

If I connect to the device in PuTTY, there are packets exchanged about the client and server protocols, then the Key Exchange Init happens, and it connects OK.

When I try to connect with Posh-SSH, the client and server protocols are exchanged, but no Key Exchange Init happens. It appears that Posh-SSH does not start the Key Exchange process.

![PuTTY to AudioCodes OK](https://user-images.githubusercontent.com/130089115/230456976-27d1464e-1ddf-4c52-b3b7-98cbd459d244.png)
![Posh-SSH to AudioCodes NOK](https://user-images.githubusercontent.com/130089115/230456992-a500c3ac-e67a-4547-8794-36baf84e890b.png)

I'd love to be able to automate processes on these systems, so would love your help in getting this connection to work!

What extra information do you need to help identify the problem?

Thanks,
Finn

**Comment by MVKozlov** [comment]
looks like #442

**Comment by finnslossdc** [comment]
Are you saying that there is a PR for Renci.SSHNet that is in conflict?

Are we waiting for the SSHNet module to be updated before these devices (AudioCodes / Some Cisco devices) will work with Posh-SSH?

**Comment by darkoperator** [comment]
Yes, im waiting for the library to fix the PR and accepts it in to their development branch before I can add it in to the module
